packages/params/src/forkName.ts

```diff
@@ -78,3 +80,16 @@ export const forkBlobs = exclude(forkAll, [ForkName.phase0, ForkName.altair, For
 export function isForkBlobs(fork: ForkName): fork is ForkBlobs {
   return isForkWithdrawals(fork) && fork !== ForkName.capella;
 }
+
+export type ForkPreElectra = ForkPreBlobs | ForkName.deneb;
+export type ForkElectra = Exclude<ForkName, ForkPreElectra>;
+export const forkElectra = exclude(forkAll, [
+  ForkName.phase0,
+  ForkName.altair,
+  ForkName.bellatrix,
+  ForkName.capella,
+  ForkName.deneb,
+]);
+export function isForkElectra(fork: ForkName): fork is ForkElectra {
+  return isForkBlobs(fork) && fork !== ForkName.deneb;
+}
```

Comment:
Can you name these `ForkPostElectra`, `forkPostElectra` and `isForkPostElectra`?

These types / values are meant to represent "all forks starting with electra and there afterwards", not just "the singular fork named electra".